src/tests/test_ocsp.cpp

```diff
@@ -225,6 +266,7 @@ class OCSP_Tests final : public Test
          results.push_back(test_response_parsing());
          results.push_back(test_response_certificate_access());
          results.push_back(test_response_verification());
+         results.push_back(test_response_verification_too_old());
```

Comment:
We should add another test that uses an OCSP response with a `next_update` specification and validate that the `max_age` does not have an effect.